components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/index.html

```diff
@@ -63,6 +63,10 @@
     <!-- SmartWizard CSS -->
     <link rel="stylesheet" type="text/css" href="editor/commons/lib/smartWizard/css/smart_wizard.css"/>
     <link rel="stylesheet" type="text/css" href="editor/css/export-docker-kubernetes.css"/>
+    <link rel="stylesheet" type="text/css" href="/editor/css/form-wizard.css" />
+
+    <script type="text/javascript" src="http://requirejs.org/docs/release/2.3.2/minified/require.js"></script>
```

Comment:
you cannot put online resources here. And why you need to add a require.js isnt it available before ?